src/components/pages/TeacherDashboard/TeacherViewRumble/TeacherViewRumbleContainer.tsx

```diff
@@ -1,29 +1,24 @@
 import React, { useEffect, useState } from 'react';
 import { useRecoilValue } from 'recoil';
 import { Prompts } from '../../../../api';
-import { useCheckBrowserState } from '../../../../hooks';
+import { useAsync, useCheckBrowserState } from '../../../../hooks';
 import { current } from '../../../../state';
-import { Loader } from '../../../common';
+import { CouldNotLoad, Loader } from '../../../common';
 import RenderTeacherViewRumble from './RenderTeacherViewRumble';
 
 const TeacherViewRumbleContainer = (): React.ReactElement => {
   const { isLoading } = useCheckBrowserState('section', 'rumble');
   const section = useRecoilValue(current.section);
   const rumble = useRecoilValue(current.rumble);
   const [prompt, setPrompt] = useState<string>();
-  const [promptIsLoading, setPromptIsLoading] = useState(true);
+
```

Comment:
This hook doesn't need a custom setter, drop the prompt state on line 13 and remove the custom setter. Get the prompt from the 3rd value in the hook